templates/api.gotmpl

```diff
@@ -252,4 +252,13 @@ func (s *Server) RunWithSigHandler(shutdown ...func() error) error {
 
 	return nil
 }
+
+// GinEngine creates a gin.Engine with generated routes
+// to make it easier to use https://github.com/appleboy/gofight
+func GinEngine(svc Service, config *Config) *gin.Engine {
```

Comment:
I think it would be simpler to just export `configureRoutes()` instead of having a very specific `GinEngine` function.

Reply:
Sure, exporting `configureRoutes()` would also be fine.
On the other hand, this whole lib is `Gin-specific`, right? So specific functions are not harmful I would say.
It uses the same signature as creating a new `Server` instance and returns a ready to use engine without exposing the information that someone has to:
- do `configureRoutes()`
- do `Server.Routes.Engine`